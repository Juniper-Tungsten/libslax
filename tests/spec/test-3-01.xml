<doc>
  <name>fish</name>
  <one>This is 'one'</one>
  <two>This is 'two'</two>
  <a>10</a>
  <b>12</b>
  <thing>hand</thing>
  <test>yup</test>
</doc>
