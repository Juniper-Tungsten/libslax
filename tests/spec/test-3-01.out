<!DOCTYPE html PUBLIC "string" "string">
<doc xmlns:axsl="http://www.w3.org/1999/XSL/Transform"><mumble><two>002</two><three>0</three></mumble><doc test10="onehand" test12="anotherhand" hand="This is 'one'">
  fish
  This is 'one'
  This is 'two'
  10
  12
  hand
  yup
</doc><name></name><one></one><two></two><a></a><b></b><thing></thing><test></test></doc>
